src/components/atoms/CampainTile/CampainTile.tsx

```diff
@@ -0,0 +1,25 @@
+import {View, Text, Image, Pressable, PressableProps} from 'react-native';
+import {styles} from './styles';
+
+type TCampainTileProps = {};
+
+export const CampainTile = ({}: TCampainTileProps) => {
+  const isImage = false;
+  const isGM = false;
+  const dupa = require('../../../static/imgs/BackgroundImage.png');
+  return (
+    <Pressable style={styles.tileWrapper}>
+      {isImage ? (
+        <Image source={dupa} style={styles.image} onError={e => console.log(e)} />
```

Comment:
Tak się obrazków w sumie nie renderuje, w sensie z palca taki placeholder to kupsko potężne jest

Każdy komponent, który powinien mieć placeholder powinien mieć to w sobie, tj. powinieneś mieć <Image/> component, który sam sobie wyrenderuje placeholder w case'ach jak np.:
source się ładuje
source nie ma
error source'a

Reply:
done